ops/jenkins/bfd-deploy-apps/Jenkinsfile

```diff
@@ -0,0 +1,340 @@
+                        gitBranchName = env.CHANGE_BRANCH
+                    } else {
+                        gitBranchName = env.BRANCH_NAME
+                    }
+                }
+            }
+        }
+
+        stage('Deploy Base') {
+            steps {
+                script {
+                    currentStage = env.STAGE_NAME
+                    try {
+                        lock(resource: lockResource) {
+                            awsAuth.assumeRole()
+                            terraform.deployTerraservice(
+                                    env: trimmedEnv,
+                                    directory: 'ops/terraform/services/base'
+                            )
+                        }
+                    } catch (org.jenkinsci.plugins.workflow.steps.FlowInterruptedException e) {
+                        currentBuild.result = 'ABORTED'
+                        sendNotifications(trimmedEnv, currentBuild.currentResult, currentStage, gitCommitId, gitRepoUrl)
+                        throw e
+                    } catch (ex) {
+                        currentBuild.result = 'FAILURE'
+                        sendNotifications(trimmedEnv, currentBuild.currentResult, currentStage, gitCommitId, gitRepoUrl)
+                        throw ex
+                    }
+                }
+            }
+        }
+
+        stage('Deploy Migrator') {
+            steps {
+                script {
+                    currentStage = env.STAGE_NAME
+                    try {
+                        def migratorAmiOverride = params.migrator_ami_override?.trim()
+
+                        lock(resource: lockResource) {
+                            migratorDeploymentSuccessful = migratorScripts.deployMigrator(
+                                amiId: migratorAmiOverride,
+                                bfdEnv: trimmedEnv,
+                                heartbeatInterval: 30, // TODO: Consider implementing a backoff functionality in the future
+                                awsRegion: awsRegion,
+                                forceDeployment: params.force_migrator_deployment
+                            )
+
+                            if (migratorDeploymentSuccessful) {
+                                println "Proceeding to Stage: 'Deploy Pipeline to ${trimmedEnv.toUpperCase()}'"
+                            } else {
+                                println "See ${migratorRunbookUrl} for troubleshooting resources."
+                                error('Migrator deployment failed')
+                            }
+                        }
+                      } catch (org.jenkinsci.plugins.workflow.steps.FlowInterruptedException e) {
+                        currentBuild.result = 'ABORTED'
+                        sendNotifications(trimmedEnv, currentBuild.currentResult, currentStage, gitCommitId, gitRepoUrl)
+                        throw e
+                    } catch (ex) {
```

Comment:
formatting seems off here - let's format the whole file if possible

Reply:
fixed